lib/chef/resource/lwrp_base.rb

```diff
@@ -89,7 +89,7 @@ def self.default_action(action_name=NULL_ARG)
           else
             action = action_name.to_sym
             @actions.push(action) unless @actions.include?(action)
-            @default_action = action
+            @default_action = [action]
```

Comment:
No, it won't be, because this section of code processes default_action only if it is a symbol.
Also the test that covers this case, already exists.